src/interactive/AccordionPanel.jsx

```diff
@@ -34,6 +34,9 @@ class AccordionPanel extends React.Component {
 	getPanelStyle(isOpen, contentEl) {
 		const style = { height: '0px' };
 
+		// console.log(`isOpen: ${isOpen}`);
```

Comment:
i recommend deleting these instead of commenting them out

Reply:
woops, missed that one